numpy/random/mtrand/mtrand.pyx

```diff
@@ -1964,8 +1971,14 @@ cdef class RandomState:
             The scale parameter, :math:`\\beta = 1/\\lambda`.
         size : int or tuple of ints, optional
             Output shape.  If the given shape is, e.g., ``(m, n, k)``, then
-            ``m * n * k`` samples are drawn.  Default is None, in which case a
-            single value is returned.
+            ``m * n * k`` samples are drawn.  If size is ``None`` (default),
+            a single value is returned if ``scale`` is a scalar.  Otherwise,
+            ``len(scale)`` samples are drawn.
```

Comment:
Done.  I prefer ```np.array(scale).size``` because it's more concise and should get the point across clear enough given the intended audience.  I had tried to describe these size-related values in more prose when working on this PR, but I couldn't come up with anything clear enough.